### PR TITLE
Introduce changes which remedy SLA alerts for SV content.

### PR DESCRIPTION
**Pull request by todor-videv1**
# Description

## What

This PR intends to remedy a problem we have with excessive SLA alerts when SV content is being published to out platform. For notifications-push if the content is SV we simply skip the check. For notifications-pull this PR adds an X-Policy that allows for reading SV content.

## Why

[UPPSF-5245](https://financialtimes.atlassian.net/browse/UPPSF-5245)

## Anything, in particular, you'd like to highlight to reviewers

Please check whether my reasoning is correct, have I deduced correctly how to handle both cases.

## Scope and particulars of this PR (Please tick all that apply)

- [x] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [ ] Feature
- [ ] Documentation
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)
___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

[UPPSF-5245]: https://financialtimes.atlassian.net/browse/UPPSF-5245?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: checks/publishCheck_test.go [hunk 1]
```diff
@@ -355,8 +379,65 @@ func TestIsCurrentOperationFinished_ContentCheck_LastModifiedDateIsEmptyStringCu
 	assert.True(t, finished, "operation should have finished successfully")
 }
 
+func TestIsCurrentOperationFinished_PushNotifications_EditorialDesk_Ignored(t *testing.T) {
```
Comment (tsvetko24) [inline checks/publishCheck_test.go:382]: Good job!

**Review by tsvetko24 — APPROVED** [review]
LGTM

**Review by antoanFT — DISMISSED** [review]

File: checks/publishCheck.go [hunk 2]
```diff
@@ -233,11 +259,25 @@ func (n NotificationsCheck) isCurrentOperationFinished(pc *PublishCheck) (operat
 
 func (n NotificationsCheck) shouldSkipCheck(pc *PublishCheck) bool {
 	pm := pc.Metric
+
+	if n.feedName == NotificationsPushFeed &&
+		(pm.EditorialDesk == CentralBankingEditorialDesk || slices.Contains(pm.Publication, SustainableViewsPublication)) {
```
Comment (tsvetko24) [inline checks/publishCheck.go:264]: What if there is SV and FT Pink Publication UUID - at the same time. There is not likely to have such use case but who knows in future...

Comment (elizhuh) [reply checks/publishCheck.go:264]: Suggestion: @tsvetko24 has a point here - the logic implemented in the other services (respectively in their OPA policies) take into consideration the case with multiple publication uuids. When both `FT Pink` and `SV` uuids are present, the content is allowed further, therefore there should be notifications about it. Maybe not a real life case but this is the current logic setup.

**Review by elizhuh — COMMENTED** [review]
Question: What about the `isTest` content? The changes are covering the `SV` and `CB` cases but I cannot see anything related with this third case.

**Review by antoanFT — APPROVED** [review]

**Pull request merged**